### PR TITLE
Minor fixes to docker

### LINKED ISSUE #4474: Feedback on page: /guides/v2.3/cloud/docker/docker-config.html
# General issue

In this instruction:

Optional: Configure the Docker global variables in the docker-compose.yaml file. For example, you can enable and configure Xdebug.

The docker-compose file name in the Magento-Cloud version 2.3 template = docker-compose.yml.

## Possible solutions:

Change docker-compose.yaml ==> docker-compose.yml

### PR DESCRIPTION
**Pull request by hguthrie**
Remove duplicate commands:
-  the ` docker-compose up -d` command was mentioned twice
-  the  `docker-compose run deploy magento-command cache:clean` command was mentioned twice

Combined the Optional copy and convert step. Made it clear that it is useful when you have a custom config file.

Fix #4474 filename typo

Removed typo from release notes

### CODE REVIEW

File: guides/v2.2/cloud/release-notes/cloud-tools.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The release notes include:
...
 
-    -   {:.new}<!-- MAGECLOUD-3251 -->—Added interactive validations for service compatibility. Now, if a requested service is incompatible with the Magento version or other services, the _interactive mode_ prompts the user with a message and a choice to continue. See the [Service versions]({{page.baseurl}}/cloud/docker/docker-config.html#service-versions) available for Docker. Use the `-n` option to skip the interactivity for CICD purposes.
+    -   {:.new}<!-- MAGECLOUD-3251 -->Added interactive validations for service compatibility. Now, if a requested service is incompatible with the Magento version or other services, the _interactive mode_ prompts the user with a message and a choice to continue. See the [Service versions]({{page.baseurl}}/cloud/docker/docker-config.html#service-versions) available for Docker. Use the `-n` option to skip the interactivity for CICD purposes.
```
Comment (meker12) [inline guides/v2.2/cloud/release-notes/cloud-tools.md]: Just curious--what if they disable interactive mode?  Would that not be recommended?

Comment (hguthrie) [reply guides/v2.2/cloud/release-notes/cloud-tools.md]: You can disable interactive on any of the CLI commands. It's useful when you don't want to be prompted. If no one is available to answer the prompt, it is of course "hanging" until it receives an answer. LOL not good for automated situation.

**Comment by hguthrie** [comment]
running tests

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**